.travis.yml

```diff
@@ -1,16 +1,27 @@
+sudo: false
 language: python
 env:
-  - REQUESTS_VERSION=2.1.0
-  - REQUESTS_VERSION=2.8.1
-  - REQUESTS_VERSION=2.9.1
-python:
-  - "2.6"
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-install:
-  - if [[ $TRAVIS_PYTHON_VERSION == 2.6* ]]; then export PYTEST_ADDOPTS='-p no:pylint'; fi
-  - pip install -r requirements.txt
-  - pip install --upgrade --force-reinstall requests==$REQUESTS_VERSION
-script: py.test
+    - TOXENV=py27-requests
```

Comment:
This seems to have gotten longer in terms of lines & execution time. Compare the last Travis build, [build#120](https://travis-ci.org/john-kurkowski/tldextract/builds/142037274), with this PR's, [build#121](https://travis-ci.org/john-kurkowski/tldextract/builds/148828810).

It's also slightly less clear which Python version Travis is running. At first glance, it looks like it's testing Python 3.5 over and over.

What's the reason for this change?